### PR TITLE
A symbol changed

### PR DESCRIPTION
**Pull request by NathanBollard**
Just a small one for more pull sharks

### CODE REVIEW

**Comment by gianlucajahn** [comment]
kinda sus

**Comment by NathanBollard** [comment]
This has been open for almost a week now. Can it be merged yet?

**Comment by matthewholt1** [comment]
Big thank you to this guy for opening a PR for a single character.

**Review by Wrongtown — CHANGES_REQUESTED** [review]
This change is unnecessary. The `(c)` is widely recognised and though unicode _is_ widely accepted, parentheses are even _more_ widely accepted 🤷‍♂️ 

Doesn't look like the repo owner is interested in merging this change mate; I suggest you should close the PR.

**Review by Johnmiicheal — COMMENTED** [review]
😂😂

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Kamil Jankowski
+Copyright © 2022 Kamil Jankowski
```
Comment (port19x) [inline LICENSE:3]: :moyai:

**Review by port19x — CHANGES_REQUESTED** [review]

**Comment by kurbus** [comment]
Okay guys this one has run its course can we close this now

**Review by MitkoVtori — CHANGES_REQUESTED** [review]
This change is unnecessary. It might look good, but it doesn't really matter. It's been a while though, @drknzz would you close it or merge it?

**Pull request closed without merging**